buildSrc/build.gradle

```diff
@@ -9,18 +9,22 @@ repositories {
 }
 
 dependencies {
-  implementation "com.android.tools.build:gradle:4.1.1"
+  implementation 'com.android.tools.build:gradle:7.0.0-beta05'
+  // See https://issuetracker.google.com/issues/176079157#comment11
+  implementation "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10"
```

Comment:
Figuring this out took literally TWO AND A HALF MONTHS. Everything else worked from the start.

Reply:
That's pretty annoying.